main/plugins/open-with-plugin.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const path = require('path');
+const {getAppsThatOpenExtension, openFileWithApp} = require('mac-open-with');
+
+const action = async context => {
+  const filePath = await context.filePath();
+  openFileWithApp(filePath, context.appUrl);
+};
+
+const apps = new Map(
+  ['mp4', 'gif', 'apng', 'webm']
+    .map(extension => [
+      extension,
+      getAppsThatOpenExtension.sync(extension).map(app => ({
+        ...app,
+        name: decodeURI(path.parse(app.url).name)
+      }))
+    ])
+    .filter(([_, apps]) => apps.length > 0)
```

Comment:
Can you sort `Gifski` to the top? People are likely to want to use it with Kap if they have it installed.